modules/cnv-disabling-fencing-on-node.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-fencing-nodes.adoc
+
+[id="cnv-disabling-fencing-on-node_{context}"]
```

Comment:
Unfortunately it seems this capability did not make it into 4.3, but I'm working to confirm that

Reply:
Alrighty, I'll comment these two out in the assembly for now